spec/services/update_fraud_matches_spec.rb

```diff
@@ -0,0 +1,47 @@
+require 'rails_helper'
+
+RSpec.describe UpdateFraudMatches do
+  let(:candidate1) { create(:candidate, id: '1', email_address: 'exemplar1@example.com') }
```

Comment:
Tiny nit. Do we need the ID here. I would've thought it would reliably instantiate them in order

Reply:
Apologies hangover from another test, didn't rmeove it because it was working reliably!